minari/dataset/minari_dataset.py

```diff
@@ -218,6 +218,25 @@ def sample_episodes(self, n_episodes: int) -> Iterable[EpisodeData]:
         episodes = self._data.get_episodes(indices)
         return list(map(lambda data: EpisodeData(**data), episodes))
 
+    def iterate_episodes(
+        self, episode_indices: Optional[List[int]] = None
+    ) -> Iterator[EpisodeData]:
+        """Iterate over episodes from the dataset.
+
+        Args:
+            episode_indices (Optional[List[int]], optional): episode indices to iterate over.
+        """
+        if episode_indices is None:
+            assert self._episode_indices is not None
+            assert self._episode_indices.ndim == 1
+            episode_indices = self._episode_indices.tolist()
+
+        for (
+            episode_index
+        ) in episode_indices:  # pyright: ignore [reportOptionalIterable]
```

Comment:
Is it possible to avoid `#pyright: ignore` here and thus make it one line?

It seems it doesn't recognize in line 140 that episode_indices can't be None, but it should work with an assert

Reply:
@Howuhh can you edit this?

Reply:
@rodrigodelazcano yup, been a little busy, I'll correct it

Reply:
done, check are all green locally, so mb this will work